include/psen_scan_v2/scanner_controller.h

```diff
@@ -64,14 +66,16 @@ class ScannerControllerT
   FRIEND_TEST(ScannerControllerTest, test_start_method_calls_correct_state_machine_event);
   FRIEND_TEST(ScannerControllerTest, test_stop_method_calls_correct_state_machine_event);
   FRIEND_TEST(ScannerControllerTest, test_udp_clients_listen_before_sending_start_request);
+  FRIEND_TEST(ScannerControllerTest, testStopRequestSending);
```

Comment:
Try to match the naming scheme to the tests above

Reply:
Hmm, but I thought we use camel case. Therefore, I suggest changing the existing tests.

Reply:
Done.